packages/proposal-sign-extension-ops/src/polyfills/i64_extend32_s.json

```diff
@@ -0,0 +1,191 @@
+{
+  "type": "Func",
+  "name": {
+    "type": "Identifier",
+    "value": "packages/proposal-sign-extension-ops/src/polyfills/i64_extend32_s/i64_extend32_s"
```

Comment:
do we need to commit the AST generated by the polyfill compilation?

Reply:
I was not sure how releases / packaging works. The proposal package directly loads the json ast to avoid unnecessary parsing. We could load and parse from WAT as suggested by Colin as an alternative